src/coupling/balance.cc

```diff
@@ -156,16 +139,35 @@ std::vector<unsigned int> Balance::add_quantities(const std::vector<string> &nam
 }
 
 
-void Balance::allocate(unsigned int n_loc_dofs,
+void Balance::allocate(const Distribution *el_ds,
```

Comment:
Can we simplify the interface of this method after I finally merged Balance cleanup pull?